bundles/org.openhab.binding.verisure/pom.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>2.5.6-SNAPSHOT</version>
```

Comment:
```suggestion
    <version>2.5.7-SNAPSHOT</version>
```

Reply:
Fixed!

Reply:
I also decided to incorporate my implementation of trigger channel, I will continue to work on that functionality but the current functionality works and gives bonus to end users.

Reply:
Do you want to incorporate your implementation in this PR or are you going to file another one after this is merged?

Reply:
I've already included it.